minjoo/app.js

```diff
@@ -5,10 +5,15 @@ const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
 const bcrypt = require("bcrypt");
-// Cost Factor
+const jwt = require("jsonwebtoken");
+
+const { validateToken } = require("./auth");
+
 const saltRounds = 12;
+const secretKey = process.env.SECRET_KEY; // (3)
 
 const { DataSource } = require("typeorm");
+const e = require("express");
```

Comment:
```suggestion
const express = require("express");
```
`e`라고 하면 `error`의 e랑 혼란이 있을 것 같아요. 항상 네이밍은 굳이 줄이지 않고 한 번에 알아 볼 수 있도록 하는 것이 좋습니다.

Reply:
헐 이거 왜 const e로 되어있죠 😂 빠르게 Commit suggestion 적용했습니다! 🫡